bin/run_unittests.sh

```diff
@@ -140,6 +140,7 @@ function run_integration_tests {
   python ../integration-tests/deploy_pyspark_models.py
   python ../integration-tests/deploy_pyspark_pipeline_models.py
   python ../integration-tests/deploy_pyspark_sparkml_models.py
+  python ../integration-tests/deploy_pytorch_models.py
```

Comment:
Since this test seems to be failing, you can revert it and I'll add it in a separate PR.